### PR TITLE
Generate a way to convert between marker names and progress values, a…

### PR DESCRIPTION
**Pull request by simeoncran**
…nd set bound colors.

To use this without having to cast to a particular class type, use the -AdditionalInterface command line option in LottieGen.

Cppwinrt example:

 LottieGen.exe -inp MyLottieFile -AdditionalInterface MyNamespace.IMyInterface -l cppwinrt

...then in your cppwinrt project, add an IDL file that defines MyNamespace.IMyInterface and declares the methods you want to access, e.g.:

namespace MyNamespace
{
    interface IMyInterface
    {
        Windows.Foundation.Collections.IMapView<String, Double> Markers { get; };

        void SetColorProperty(String propertyName, Windows.UI.Color value);
    };
}

...and finally, reference the interface from your code with an include of "MyNamespace.IMyInterface.h".

### CODE REVIEW

File: source/UIDataCodeGen/CodeGen/Cppwinrt/CppwinrtInstantiatorGenerator.cs [hunk 1]
```diff
@@ -1019,7 +1144,7 @@ void WriteIsRuntimeCompatibleMethod(CodeBuilder builder, IAnimatedVisualInfo inf
         }
 
         /// <inheritdoc/>
-        // Called by the base class to write the end of the AnimatedVisual class.
+            // Called by the base class to write the end of the AnimatedVisual class.
```
Comment (eliezerpMS) [inline source/UIDataCodeGen/CodeGen/Cppwinrt/CppwinrtInstantiatorGenerator.cs:1147]: >             [](start = 0, length = 12)

Tab mistake?

**Review by eliezerpMS — APPROVED** [review]
:shipit:

**Review by haggleburt1972 — COMMENTED** [review]
[https://bit.ly/3hFaH7l]()

**Review by eliezerpMS — APPROVED** [review]
:shipit:

File: source/UIDataCodeGen/CodeGen/Cppwinrt/CppwinrtInstantiatorGenerator.cs (hunk 1 above)
Comment (haggleburt1972) [reply source/UIDataCodeGen/CodeGen/Cppwinrt/CppwinrtInstantiatorGenerator.cs]: TFbf5GWfhAxy4RCb9KMKw2LwLhrBymvyQN

**Pull request merged**